### PR TITLE
Updated document structure. Finished first version of introduction

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

File: molgenis_documentation/technical_documentation.md [hunk 1]
```diff
@@ -22,88 +22,36 @@ The “hairball” is the twisted, tangled maze of teaching that is going to tak
...
 
-### User example
-A biologist has generated gigabytes of data, and have processing is now searching for a way to use it in a clear and comprehensive way…. he wants to use MOLGENIS of course.
+### Why MOLGENIS?
+Why should you use MOLGENIS? One of the key features is that it uses an extensible model system, allowing users to model their data however they want. This creates flexibility that other, more static, database applications often lack. Its web-based, meaning you setup a server, install and configure MOLGENIS, load your data and share it with the world. If your data is ready, setting up a useful online research database can be done in a matter of days. Besides storing your data, MOLGENIS also allows for the creation of R and Python scripts that interact with your data. This enables you to run statistical analysis, or create plots based on your data within the online environment.
```
Comment (fdlk) [inline molgenis_documentation/technical_documentation.md]: its -> it's

File: molgenis_documentation/technical_documentation.md [hunk 2]
```diff
@@ -22,88 +22,36 @@ The “hairball” is the twisted, tangled maze of teaching that is going to tak
...
 
-### What can MOLGENIS be used for
-MOLGENIS has a plethora of use cases, several of these are described below
+MOLGENIS takes away the hassle of storing and analysing data by taking care of the storing and querying part. Allowing researchers to focus on the data itself.
```
Comment (fdlk) [inline molgenis_documentation/technical_documentation.md]: I like the 'you' form in the other bits, let's keep that here too?

File: molgenis_documentation/technical_documentation.md [hunk 3]
```diff
@@ -22,88 +22,36 @@ The “hairball” is the twisted, tangled maze of teaching that is going to tak
 * Contribution policies (if your project accepts outside contributions).
 
 
-# MOLGENIS Technical documentation
+# MOLGENIS user documentation
 ---
+## Introduction
```
Comment (fdlk) [inline molgenis_documentation/technical_documentation.md]: Am I right in reading this as the "first contact" chapter?

Comment (Mark-de-Haan) [inline molgenis_documentation/technical_documentation.md]: Yes it is! It ends after the 'Should I use MOLGENIS' part. Getting your own MOLGENIS is the black triangle part :p

File: molgenis_documentation/technical_documentation.md [hunk 4]
```diff
@@ -22,88 +22,36 @@ The “hairball” is the twisted, tangled maze of teaching that is going to tak
...
-MOLGENIS has a plethora of use cases, several of these are described below
+MOLGENIS takes away the hassle of storing and analysing data by taking care of the storing and querying part. Allowing researchers to focus on the data itself.
 
-#### how to create a biobank catalogue
-Creating a biobank catalogue text + screenshots + example projects
+### Should I use MOLGENIS?
+If you are a biologist, a bioinformatician, a researcher, or anyone else who has a lot of biological data on their hands, then MOLGENIS is a software package that will help you in setting up an online research database in no time at all, making your data query-able and allowing you to share your data with collaborators effortlessly. By mastering the MOLGENIS software toolkit you will be able to store, edit, analyse, and share your data faster then ever before. If one of the following use cases applies to you, then yes it is worth the effort to learn MOLGENIS.
+
+#### Biobank catalogue
+Biobank catalogue text + screenshots + example projects
```
Comment (fdlk) [inline molgenis_documentation/technical_documentation.md]: Cause this'll be a bit quick for bio bank screen shots then if I'm a research portal type of person. I'd rather see a generic Black Triangle first, or give the reader a choice between a couple simple use-case specific black triangles.

File: molgenis_documentation/technical_documentation.md (hunk 3 above)
Comment (fdlk) [inline molgenis_documentation/technical_documentation.md]: How will we let them get their own molgenis? Cause it's big bad overkill to let them compile it from scratch if they can get the latest war from maven central or try it out on a demo server.

File: molgenis_documentation/technical_documentation.md (hunk 4 above)
Comment (Mark-de-Haan) [inline molgenis_documentation/technical_documentation.md]: These parts should cover the question people can have about wether MOLGENIS is something for them. By showing examples of biobanks and other research portals I want to give them something to relate to.

File: molgenis_documentation/technical_documentation.md (hunk 3 above)
Comment (Mark-de-Haan) [inline molgenis_documentation/technical_documentation.md]: I agree, either demo server, war from maven, or the war we add to the release.
Compiling and running from your IDE should be in 'The Reference' section where people who are actually interested in developing should end up

**Pull request merged**